conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, CMake, tools
+
+
+class PahomqttclibConan(ConanFile):
+    name = "paho_mqtt_c"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Elearcommonlib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        if (self.settings.os == "Android"):
+            cmake.definitions["Platform"] = "android"
+        cmake.configure(source_folder=".")
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="src")
+        self.copy("*", dst="lib", src="lib"keep_path=False)
```

Comment:
Does this also copy header files?

Reply:
Missing ", " before keep_path

Reply:
> Does this also copy header files?
No, It wont copy header files into lib directory, it checks for `lib directory files in build directory` to   
`lib directory in package`

Reply:
> Missing ", " before keep_path
handled